diff --git a/node_modules/react-native-agora/android/build.gradle b/node_modules/react-native-agora/android/build.gradle
index 8e41e96..30813a0 100644
--- a/node_modules/react-native-agora/android/build.gradle
+++ b/node_modules/react-native-agora/android/build.gradle
@@ -167,6 +167,7 @@ dependencies {
   /// dependencies start
   api 'io.agora.rtc:full-sdk:4.6.4'
   implementation 'io.agora.rtc:full-screen-sharing:4.6.4'
+  implementation 'io.agora.rtc:clear-vision:4.6.4'
   api 'io.agora.rtc:iris-rtc:4.6.4-build.6'
   /// dependencies end
 }
